IronSoftware.Drawing/IronSoftware.Drawing.Common/CropRectangle.cs

```diff
@@ -1,15 +1,36 @@
+using System;
+
 namespace IronSoftware.Drawing
 {
+    // <summary>
+    /// <para>A universally compatible Rectangle for .NET 7 and .NET 6, .NET 5, .NET Core. Windows, NanoServer, IIS,  macOS, Mobile, Xamarin, iOS, Android, Google Compute, Azure, AWS and Linux compatibility.</para>
+    /// <para>Works nicely with popular Image Rectangle such as System.Drawing.Rectangle, SkiaSharp.SKRect, SixLabors.ImageSharp.Rectangle, Microsoft.Maui.Graphics.Rect.</para>
+    /// <para>Implicit casting means that using this class to input and output Rectangle from public API's gives full compatibility to all Rectangle type fully supported by Microsoft.</para>
+    /// </summary>
     public partial class CropRectangle
     {
+        /// <summary>
+        /// Construct a new CropRectangle.
+        /// </summary>
+        /// <seealso cref="CropRectangle"/>
         public CropRectangle() { }
 
-        public CropRectangle(int x, int y, int width, int height)
+        /// <summary>
+        /// Construct a new CropRectangle.
+        /// </summary>
+        /// <param name="x">The x-coordinate of the upper-left corner of this Rectangle</param>
+        /// <param name="y">The y-coordinate of the upper-left corner of this Rectangle</param>
+        /// <param name="width">The width of this Rectangle</param>
+        /// <param name="height">The height of this Rectangle</param>
+        /// <param name="units">The unit of measurement of this Rectangle</param>
```

Comment:
Should we change:
`        /// <param name="units">The unit of measurement of this Rectangle</param>`
To:
`        /// <param name="units">The unit of measurement of width and height for this Rectangle</param>`

Reply:
It'll convert x, and y as well.
So, I think it should be
`/// <param name="units">The measurement unit of this Rectangle</param>`

What do you think?

Reply:
Looks good.